guides/_posts/2014-01-5-vfl.html

```diff
@@ -300,34 +301,144 @@
-/* VFL */
-@horizontal |-24-[#find]-[#findNext]-[#findField(>=30)]-24-| in(#popup);
+@v | (.featured > article ~ .title) | in(#elm > .class .class2"_fallout")
+{% endhighlight %}
+
+<a name="vflsplats"></a>
+## VFL Splat
+When a selector in your VFL expression finds more than one element, due to [plural binding](/guides/selectors/#pluralbinding),
+if the number of element within each selectors expression are not equal, then some elements will not be constraint.
+<a href="http://codepen.io/cbchouinard/pen/vEJqbG" target="_blank">Live example.</a>
+
+In this example, the last two `article` elements will not be constrained since there's no `section` element to bind to.
+As you can see in the CCSS generated constraints, the elements are pluraly binded.
+
+{% highlight css %}
+
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<footer></footer>
+
+<style type="text/gss">
+
+  /* the last two articles will not be constraints because of plural binding */
+  @h |-(section)-(article)-(footer)-| in(::window);
+
+  /* this generates the following CCSS */
+  ::window[left] + [hgap] == section[left];
+  section[right] + [hgap] == article[left]; /* plural binding */
+  article[right] + [hgap] == footer[left]; /* plural binding */
+  footer[right] + [hgap] == ::window[right];
+
+</style>
+
+{% endhighlight %}
+
+If the number of selected elements matches, the position of the elements within each collection of selected
+elements will overlap since you're basically constraining the whole collection to the same values. <a href="http://codepen.io/cbchouinard/pen/yyodWx" target="_blank">Live example.</a>
+
+{% highlight html %}
+
+<section></section>
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<footer></footer>
+
+<style type="text/gss">
+
+/* the 3 sections will overlap over each other as the 3 articles. */
+@h |-(section)-(article)-(footer)-| in(::window);
+
+</style>
+
+{% endhighlight %}
+
+Splats will iterate over the collection of selected elements and position elements
+next to each other. <a href="http://codepen.io/cbchouinard/pen/ogerrX" target="_blank">Live example.</a>
+
+{% highlight css %}
+
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<div></div>
+<div></div>
+
+<style type="text/gss">
+
+/* elements are not 'pluraly' binded */
+@h |-(section)-10-...-(article)-20-...-(div)...-| in(::window);
+
+</style>
 
-/* Equivalent CCSS */
-#popup[left] + 24 == #find[left];
-#find[right] + [hgap] == #findNext[left];
-#findField[width] >= 30;
-#findNext[right] + [hgap] == #findField[left];
-#findField[right] + 24 == #popup[right];
 {% endhighlight %}
 
+In the previous example, the two `section` will be positioned next to each other with a gap of 10px. The
+three articles will be positioned next to each other with a gap of 20px. Finally the two div will be right next to each
+other without any gap.
```

Comment:
There's some inconsistency with singular/plural here and how elements are being referenced.

We could either say:
* 1
  - two `section`s
  - three `article`s
  - two `div`s
* 2
  - two sections
  - three articles
  - two divs
* 3
  - two `section` elements
  - three `article` elements
  - two `div` elements

I'd probably go with option 2 or 3.